frontend/src/components/pages/RegistrantExperience/PersonalInfo/index.tsx

```diff
@@ -1,26 +1,110 @@
 import React from "react";
-import { Box, Checkbox, Text } from "@chakra-ui/react";
+import { SmallAddIcon } from "@chakra-ui/icons";
+import { Box, Button, Divider, Text, useToast, VStack } from "@chakra-ui/react";
+import { RegistrantExperienceCamper } from "../../../../types/CamperTypes";
+import { usePersonalInfoHook } from "./personalInfoReducer";
+import CamperCard from "./CamperCard";
+import { PersonalInfoActions } from "../../../../types/PersonalInfoTypes";
+import ContactCard from "./ContactCard";
+import { CampSession } from "../../../../types/CampsTypes";
 
 type PersonalInfoProps = {
-  isChecked: boolean;
-  toggleChecked: () => void;
+  nextBtnRef: React.RefObject<HTMLButtonElement>;
+  campers: RegistrantExperienceCamper[];
+  campName: string;
+  campSessions: CampSession[];
+  setCampers: React.Dispatch<
+    React.SetStateAction<RegistrantExperienceCamper[]>
+  >;
+};
+
+const checkSpaceAvailable = (
+  campSessions: CampSession[],
+  campers: RegistrantExperienceCamper[],
```

Comment:
I'm a little confused over what you mean by `something to differentiate it from regular campers already registered`. To preface, I used the `RegistrantExperienceCamper` to define the data type of campers throughout the 4 registrant experience steps. Since `RegistrantExperienceCamper` is specific only to the `RegistrantExperience` folder, I'm not sure where the need to differentiation is needed.